systemtest/src/test/java/io/strimzi/systemtest/kafka/listeners/MultipleListenersST.java

```diff
@@ -94,6 +135,16 @@ void testCombinationOfInternalAndExternalListeners() {
     @Tag(LOADBALANCER_SUPPORTED)
     @IsolatedTest("Using more tha one Kafka cluster in one namespace")
     @Tag(EXTERNAL_CLIENTS_USED)
+    @TestDoc(
+        description = @Desc("Test verifying the behavior of multiple load balancers in a single namespace using more than one Kafka cluster."),
+        steps = {
+            @Step(value = "Run listeners test with LOADBALANCER type.", expected = "Listeners test executes successfully with load balancers."),
+            @Step(value = "Validate the results.", expected = "Results match the expected outcomes for multiple load balancers.")
+        },
```

Comment:
I think it would be good to unify the naming for LB listeners

Reply:
Maybe even across all listeners

Reply:
Sure, I have updated 
```
7. NodePort
8. LoadBalancer
9. Route
10. Internal
11. External
```